### PR TITLE
Docs: Tiny update to Async Action Creators example

### PR DESCRIPTION
**Pull request by meenie**

### CODE REVIEW

**Comment by timdorr** [comment]
I'm not sure this really does anything, but ¯\\\_(ツ)\_/¯

**Comment by meenie** [comment]
The code below it was referencing a `reddit` variable which didn't exist.  I saw that later in the docs the code is used inside a middleware where that variable did exist.

File: docs/advanced/AsyncActions.md [hunk 1]
```diff
@@ -301,9 +301,10 @@ import { createStore } from 'redux';
 import { selectReddit, requestPosts, receivePosts } from './actions';
 import rootReducer from './reducers';
 
+const reddit = 'reactjs';
 const store = createStore(rootReducer);
 
-store.dispatch(selectReddit('reactjs'));
+store.dispatch(selectReddit(reddit));
 
 store.dispatch(requestPosts('reactjs'));
```
Comment (gaearon) [inline docs/advanced/AsyncActions.md]: Missed this one ^^^
(Not a problem, rearranging that section anyway)

Comment (meenie) [inline docs/advanced/AsyncActions.md]: Woops! Sounds good :)

**Pull request merged**